<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFGWadseekerIdgames</class>
 <widget class="QWidget" name="CFGWadseekerIdgames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>419</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="cbUseIdgames">
     <property name="text">
      <string>Use Idgames archive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameWithContent">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Idgames archive URL (changing this is NOT recommended):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="leIdgamesURL"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnIdgamesURLDefault">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../media/resources.qrc">
            <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../media/resources.qrc"/>
 </resources>
 <connections/>
</ui>
